### DISCUSSION TITLE
学习imcoding 《签名者 Signer》章节，线上环境，run课程代码报错 “use Std::Signer;  Invalid 'use'. Unbound module: '(Std=0x1)::Signer'”

### DISCUSSION THREAD

**Discussion by baidang201** [NonceGeek/Move-Camp#236] [Q&A] [answered]
我看课程的代码例子，有"use Std::Signer; "，完全复制下来的居然报错

https://imcoding.online/courses/move-language/learn?lesson=1&chapter=3

script {
  use Std::Signer;

  fun main(s: signer) {
    assert!(Signer::address_of(&s) == @0x42, 0);
  }
}

**Comment by Demian101** [comment, marked as answer]
https://github.com/psyljz/imcoding-move-reference-answer/blob/main/C2_%E7%BB%93%E6%9E%84%E4%BD%93%E5%92%8C%E8%B5%84%E6%BA%90/1_%E7%BB%93%E6%9E%84%E4%BD%93.md

这边有些参考答案

**Reply by baidang201** [reply]
答案是ok的，imcoding的运行环境有问题，这个标准答案拷过去也报错
